tests/modules/configs/test_config.py

```diff
@@ -121,3 +120,134 @@ def test_get_conf_path_file_not_found(self):
             conf.get_conf_path(conf_path)
         assert ex.type == FileNotFoundError
         assert err_msg.get_conf_path.get("file_not_found") in str(ex.value)
+
+
+class TestMergedConf:
+    """Class to test functions of MergedConf Class"""
+
+    @staticmethod
+    def none_to_dict(obj):
+        if obj is None:
+            return {}
+        return obj
+
+    @pytest.mark.parametrize("dct, expect", [
+        [merge_conf.dct1, merge_conf.expect1],
+        [merge_conf.dct2, merge_conf.expect2],
+        [merge_conf.dct3, merge_conf.expect3],
+    ])
+    def test_merge_conf(self, dct, expect):
```

Comment:
This test is alright, although you could define more meaningful variable names (top-level only i.e. `dct1`, `expect1` etc.).